### DISCUSSION TITLE
osu!lazer ignores hyprland tablet settings

### DISCUSSION THREAD

**Discussion by arsenez2006** [ppy/osu#29598] [Q&A] [answered]
I am using Hyprland to configure my tablet's area.
Here is the config:
```
device {
  name = wacom-bamboo-connect-pen
  active_area_size = 65 36.56
  active_area_position = 24.23 17.77
}
```
The numbers were taken from my old OTD config, but due to switching to another hardware, which is not compatible with OTD, I have to configure my tablet through Hyprland. It works just fine in normal environment, but osu!lazer maps my tablet to full area even though in-game driver is disabled. I also tried to switch between display modes, but nothing changed, except I could see the difference between my real cursor and in-game cursor positions.

**Comment by arsenez2006** [comment]
O'key, I figured it out, for some reason, switching to fullscreen causes this issue, but swithing back to borderless without quitting the game does not fix this. Should I open a new issue?

**Comment by Stoppedpuma** [comment, marked as answer]
This is an issue on Hyprlands end, probably related to [this](https://github.com/hyprwm/Hyprland/issues/6023) or [Hyprland using the applications resolution for mapping tablet areas](https://github.com/hyprwm/Hyprland/issues/7481).
